pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/TopicsImpl.java

```diff
@@ -1578,6 +1578,41 @@ public void failed(Throwable throwable) {
         }
     }
 
+    @Override
+    public Long getBacklogSizeByMessageId(String topic, String messageId, boolean checkLedgerExists)
```

Comment:
There is a `MessageId` object we should use that instead of a string

Reply:
Can I use a PUT method here? If MessageId object used